devnet/tests.yaml

```diff
@@ -13,7 +13,7 @@ spec:
           command:
             - /bin/sh
             - -c
-            - "sh /app/testing/sdk.sh && touch /app/testing/success"
+            - "bash /app/testing/sdk.sh && touch /app/testing/success"
```

Comment:
This command invokes `/bin/sh`, should that be `/bin/bash` instead? What's the motivation behind this change?

Reply:
without this change, here is what happens: https://github.com/wormhole-foundation/wormhole/actions/runs/5189716428/jobs/9355110068

(The SDK-ci-tests container starts up way too early)

Reply:
Oh also currently, this emulates how the readiness check for 'accountant-ci-tests' is done right below in this file